docs/source/content/debug.rst

```diff
@@ -0,0 +1,87 @@
+A Short Debugging Guide
+-----------------------
+
+We have put together a list of common traps you may fall into, hopefully you will find the tools here to get
+yourself out of them!
+
+* Did you forget `conda activate sharpy_env` and `source bin/sharpy_vars.sh`?
+
+    - If you do in the terminal: `which sharpy`, do you get the one you want?
+    - If you do `which python`, does the result point to `anaconda3/envs/sharpy_env/bin` (or similar)?
+
+* Wrong input (inconsistent connectivities, mass = 0...)
+
+    * Sometimes not easy to detect. For the structural model, run `BeamLoader` and `BeamPlot` with no structural solver
+      in between. Go over the structure in Paraview. Check the `fem.h5` file with HDFView.
+    * Remember that connectivities are ordered as $[0, 2, 1]$ (the central node goes last).
+    * Make sure the `num_elem` and `num_node` variables are actually your correct number of elements and nodes.
+
+* Not running the actual case you want to.
+
+    * Cleanup the folder and regenerate the case
+
+* Not running the SHARPy version you want.
+
+    * Check at the beginning of the execution the path to the SHARPy folder.
+
+* Not running the correct branch of the code.
+
+    * You probably want to use `develop`. Again, check the first few lines of SHARPy output.
+
+* Very different (I'm talking orders of magnitude) stiffnesses between nodes or directions?
+
+* The UVLM requires a smaller vortex core cutoff?
```

Comment:
Can you remove this one? We don't have a vortex cutoff for the nonlinear UVLM anymore.
```suggestion
* Maybe the UVLM requires a smaller a smaller vortex core cutoff (only for linear UVLM simulations, as the nonlinear uses another vortex core model).
```

Reply:
Speaking about it, how difficult would it be to adapt the linear UVLM to the epsilon core model instead of the cutoff, or at least having the option to choose?

Reply:
> Speaking about it, how difficult would it be to adapt the linear UVLM to the epsilon core model instead of the cutoff, or at least having the option to choose?

Good idea, I'm including this in #48